js/src/forum/utils/KeyboardNavigatable.ts

```diff
@@ -51,20 +51,35 @@ export default class KeyboardNavigatable {
   /**
    * Provide a callback to be executed when the current item is selected..
    *
-   * This will be triggered by the Return and Tab keys..
+   * This will be triggered by the Return key.
    */
   onSelect(callback: KeyboardEventHandler): KeyboardNavigatable {
     const handler: KeyboardEventHandler = (e) => {
       e.preventDefault();
       callback(e);
     };
 
-    this.callbacks.set(9, handler);
```

Comment:
This is technically a breaking change as it changes behavior, is it not? Not sure how we want to handle it, if at all.

Reply:
It is, yes. We *could* add a second param to override the original behaviour, which might be a better idea.

Reply:
I'm in favor of replacing the second param with a diff method for 2.0. Just want to void breaking outside of major releases.